### PR TITLE
Move config from XML to PHP

### PR DESCRIPTION
**Pull request by wbloszyk**
Move configuration to PHP.

### CODE REVIEW

File: src/Resources/config/blocks.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+use Sonata\SeoBundle\Block\Breadcrumb\HomepageBreadcrumbBlockService;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ReferenceConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // Use "service" function for creating references to services when dropping support for Symfony 4.4
+    // Use "param" function for creating references to parameters when dropping support for Symfony 5.1
+    $containerConfigurator->parameters()
+
+        ->set('sonata.seo.block.breadcrumb.homepage.class', HomepageBreadcrumbBlockService::class);
```
Comment (core23) [inline src/Resources/config/blocks.php:23]: We should remove all class parameters

Comment (wbloszyk) [reply src/Resources/config/blocks.php:23]: I just thinking. Mybe we should also do it in `2.x`?

Comment (core23) [reply src/Resources/config/blocks.php:23]: IMHO it's a breaking change if we remove this on the stable branch.

Comment (wbloszyk) [reply src/Resources/config/blocks.php:23]: Move config to PHP in `2.x` and remove parameters in `3.x`.

**Comment by core23** [comment]
Done with #604

**Pull request closed without merging**